packages/web-react/src/types/link.ts

```diff
@@ -17,7 +25,9 @@ export interface LinkBaseProps<C = void> extends ChildrenProps, StyleProps, Tran
   /** Color of the Link */
   color?: ActionLinkColorsDictionaryType<C>;
   /** Whether is the Link underlined */
+  /** @deprecated "isUnderlined" property will be replaced in the next major version. Please use "underline" instead. */
   isUnderlined?: boolean;
+  underlined?: UnderlinedOptions;
```

Comment:
⚠️ is it underline or underlined? In twig you use underline

Reply:
I changed it to `underline`, but it might be confusing to use underline in some places and `underlined` in others. I might change it back to `underlined` and update it everywhere to use this name.

Reply:
renamed to `underlined`